Source/Charts/Renderers/HorizontalBarChartRenderer.swift

```diff
@@ -628,4 +655,31 @@ open class HorizontalBarChartRenderer: BarChartRenderer
     {
         high.setDraw(x: barRect.midY, y: barRect.origin.x + barRect.size.width)
     }
+
+    override internal func createBarPath(for rect: CGRect, roundedCorners: UIRectCorner) -> UIBezierPath {
```

Comment:
could bar renderer and horizontal bar renderer share the same `createBarPath` and `findMostLeftRectInBar` somehow?